pydecred/rpc.py

```diff
@@ -64,6 +64,137 @@ def getBlockchainInfo(self):
         """
         return GetBlockChainInfoResult.parse(self.call("getblockchaininfo"))
 
+    def getGenerate(self):
+        """
+        Returns if the server is set to generate coins (mine) or not.
+
+        Returns:
+            bool: True if mining, false if not.
+        """
+        return self.call("getgenerate")
+
+    def getHashesPerSec(self):
+        """
+        Returns a recent hashes per second performance measurement while
+        generating coins (mining).
+
+        Returns:
+            int: The number of hashes per second.
+        """
+        return self.call("gethashespersec")
+
+    def getHeaders(self, blockLocators, hashStop):
+        """
+        Returns block headers starting with the first known block hash from the
+        request.
+
+        Args:
+            blockLocators (list(string)): Array of block locator hashes.
+                Headers are returned starting from the first known hash in this
+                list.
+            hashStop (string): Block hash to stop including block headers for.
+                Set to zero to get as many blocks as possible.
+
+        Returns:
+            list(string): Serialized block headers of all located blocks,
+                limited to some arbitrary maximum number of hashes (currently
+                2000, which matches the wire protocol headers message, but this
+                is not guaranteed).
+        """
+        return self.call("getheaders", blockLocators, hashStop)["headers"]
+
+    def getInfo(self):
+        """
+        Returns a JSON object containing various state info.
```

Comment:
~~JSON object~~ Most of the time it works to steal the description directly from dcrctl, but it doesn't work in this case. Same for `getMiningInfo`, `getNetTotals`, and `getNetworkInfo`.